docs/src/components/DocsPageBuilder/BpkMarkdownRenderer.js

```diff
@@ -40,8 +40,8 @@ import STYLES from './BpkMarkdownRenderer.scss';
 
 const getClassName = cssModules(STYLES);
 
-const TAG_NAMES = ['h1', 'h1', 'h2', 'h3'];
-const TEXT_STYLES = ['xxl', 'xxl', 'xl', 'lg'];
+const TAG_NAMES = ['h1', 'h1', 'h2', 'h3', 'h4'];
+const TEXT_STYLES = ['xxl', 'xxl', 'xl', 'lg', 'base'];
```

Comment:
I know not part of this but should we add our other supported sizes to this list? `md`, `sm`, `xs`?

Reply:
Well the heading level should only be 1-4

Reply:
Ok thats fine, just curious mainly as our `BpkText` component accepts different text styles but that might be another discussion 😄